indico/migrations/versions/20171124_1138_2af245be72a6_review_questions_models.py

```diff
@@ -0,0 +1,65 @@
+    op.add_column('abstract_review_questions',
+                  sa.Column('is_required', sa.Boolean(), nullable=False, server_default='true'),
+                  schema='event_abstracts')
+    op.alter_column('abstract_review_questions', 'is_required', server_default=None, schema='event_abstracts')
+    op.add_column('abstract_review_questions', sa.Column('field_data', sa.JSON(), nullable=False, server_default='{}'),
+                  schema='event_abstracts')
+    op.alter_column('abstract_review_questions', 'field_data', server_default=None, schema='event_abstracts')
+    op.execute('ALTER TABLE event_abstracts.abstract_review_ratings ALTER COLUMN "value" TYPE JSON '
+               'USING to_json(value)')
+
+    op.add_column('review_questions',
+                  sa.Column('field_type', sa.String(), nullable=False, server_default='rating'),
+                  schema='event_paper_reviewing')
+    op.alter_column('review_questions', 'field_type', server_default=None, schema='event_paper_reviewing')
+    op.add_column('review_questions',
+                  sa.Column('is_required', sa.Boolean(), nullable=False, server_default='true'),
+                  schema='event_paper_reviewing')
+    op.alter_column('review_questions', 'is_required', server_default=None, schema='event_paper_reviewing')
+    op.add_column('review_questions', sa.Column('field_data', sa.JSON(), nullable=False, server_default='{}'),
+                  schema='event_paper_reviewing')
+    op.alter_column('review_questions', 'field_data', server_default=None, schema='event_paper_reviewing')
+    op.execute('ALTER TABLE event_paper_reviewing.review_ratings ALTER COLUMN "value" TYPE JSON '
+               'USING to_json(value)')
+
+
+def downgrade():
+    op.execute("DELETE FROM event_abstracts.abstract_review_ratings WHERE question_id "
+               "IN(SELECT id FROM event_abstracts.abstract_review_questions WHERE field_type != 'rating')")
+    op.execute("DELETE FROM event_abstracts.abstract_review_questions WHERE field_type != 'rating'")
+    op.execute('ALTER TABLE event_abstracts.abstract_review_ratings ALTER COLUMN "value" TYPE INT '
+               'USING to_json(value)::TEXT::INT')
+    op.drop_column('abstract_review_questions', 'field_type', schema='event_abstracts')
+    op.drop_column('abstract_review_questions', 'is_required', schema='event_abstracts')
+    op.drop_column('abstract_review_questions', 'field_data', schema='event_abstracts')
+    op.execute("DELETE FROM event_paper_reviewing.review_ratings WHERE question_id "
+               "IN(SELECT id FROM event_paper_reviewing.review_questions WHERE field_type != 'rating')")
+    op.execute("DELETE FROM event_paper_reviewing.review_questions WHERE field_type != 'rating'")
+    op.execute('ALTER TABLE event_paper_reviewing.review_ratings ALTER COLUMN "value" TYPE INT '
+               'USING to_json(value)::TEXT::INT')
```

Comment:
Why `to_json()`? value is already JSON here, isn't it?

Reply:
Not sure why.. my overlooking